<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find rex_ground_calib)/launch/custom_rosconsole.conf"/>
	<node pkg="rex_ground_calib" type="rex_ground_calib" name="rex_ground_calib" output="screen">
		<param name="input_cloud" value="/camera/point_cloud/cloud"/>
    <param name="odom_frame" value="/odom"/>
		<param name="camera_frame" value="/zed_initial_frame"/>
    <param name="floor_tolerance" value="0.0750"/>

    <param name="base_offset_x" value="0.075"/>
    <param name="base_offset_y" value="0.25"/>
    <param name="base_offset_z" value="0.05"/>
	</node>
</launch>
